packages/devtools_app/lib/src/shared/memory/adapted_heap_object.dart

```diff
@@ -7,24 +7,36 @@ import 'package:vm_service/vm_service.dart';
 import 'class_name.dart';
 import 'simple_items.dart';
 
-/// Names for json fields.
-class _JsonFields {
-  static const String code = 'code';
-  static const String references = 'references';
-  static const String klass = 'klass';
-  static const String library = 'library';
-  static const String shallowSize = 'shallowSize';
+// TODO: remove wrappers.
```

Comment:
Is this to be done in a future change? If so, can we leave a comment explaining why they're here?

Reply:
Oh, forgot to remove. Thanks. It improved memory even more.